sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AbstractSqlParser.scala

```diff
@@ -91,6 +91,20 @@ abstract class AbstractSqlParser extends AbstractParser with ParserInterface {
     }
   }
 
+  /** Creates BatchBody for a given SQL string. */
+  override def parseBatch(sqlText: String): BatchBody = parse(sqlText) { parser =>
```

Comment:
I kept `sqlText` as a param name everywhere, but maybe we can change it to something else?
Some options:
- `batchSqlText`
- `bodyText`
- `batchBodyText`

Reply:
+1 for batchBodyText